dockers/docker-platform-monitor/Dockerfile

```diff
@@ -5,7 +5,21 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update
 
-RUN apt-get install -y sensord
+RUN apt-get install -y sensord monit
+
+## Config monit
+RUN sed -i 's/^\s*set daemon/# set daemon/' /etc/monit/monitrc
+RUN sed -i 's/^# set logfile syslog/set logfile syslog/' /etc/monit/monitrc
+RUN sed -i 's/^\s*set logfile \/var/# set logfile \/var/' /etc/monit/monitrc
+RUN sed -i 's/^# set httpd port/set httpd port/' /etc/monit/monitrc
+RUN sed -i 's/^#    use address localhost/   use address localhost/' /etc/monit/monitrc
+RUN sed -i 's/^#    allow localhost/   allow localhost/' /etc/monit/monitrc
+RUN sed -i 's/^#    allow admin:monit/   allow admin:monit/' /etc/monit/monitrc
+RUN sed -i 's/^#    allow @monit/   allow @monit/' /etc/monit/monitrc
+RUN sed -i 's/^#    allow @users readonly/   allow @users readonly/' /etc/monit/monitrc
+
+RUN echo "check filesystem root-aufs with path /" >> /etc/monit/monitrc
+RUN echo "  if space usage > 85% for 5 times within 15 cycles then alert" >> /etc/monit/monitrc
```

Comment:
Technically they are the same. The benefit of sed is reusing the default config, and easy to detect incompatibility issue in future monit config (which may be rare but possible).

---
In reply to: [115619051](https://github.com/Azure/sonic-buildimage/pull/582#discussion_r115619051) [](ancestors = 115619051)